lib/src/rules/iterable_contains_unrelated_type.dart

```diff
@@ -13,10 +13,12 @@ const _desc = r'Invocation of Iterable<E>.contains with references of unrelated'
 
 const _details = r'''
 
-**DON'T** Invoke `contains` on `Iterable` with an instance of different type than
-the parameter type since it will invoke `==` on its elements and most likely will
+**DON'T** invoke `contains` on `Iterable` with an instance of different type than
+the parameter type.
+
+Doing this will invoke `==` on its elements and most likely will
 return `false`. Strictly speaking it could evaluate to true since in Dart it
-is possible for an Iterable to contain elements of type unrelated to its
+is possible for an `Iterable` to contain elements of type unrelated to its
 parameter type, but this practice also should be avoided.
```

Comment:
I believe that the last statement is not true in strong mode, and hence should be removed.